security/tests/agent/sdsc.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sdsc
+  labels:
+    app: sdsc
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        "sidecar.istio.io/inject": "false"
+      labels:
+        app: sdsc
+    spec:
+      volumes:
+      - hostPath:
+          path: /var/run/sds/uds_path
+          type: Socket
+        name: sds-uds-path
+      containers:
+      - name: client
+        image: gcr.io/jianfeih-test/sdsclient:dev
```

Comment:
is this right ?

Reply:
not ideal for sure... but we don't have public image produced yet. changed to istio-testing image.